docs/src/man/syntax.md

```diff
@@ -148,6 +148,46 @@ Order = [:type]
 ```
 ````
 
+You may have some names in your package that are unexported but are still part
+of your package's public API. You can tell Documenter about these names by
+defining the `NONEXPORTED_PUBLIC_NAMES` vector in your package. The names in
+`NONEXPORTED_PUBLIC_NAMES` will be treated as public by Documenter. Example
+usage:
+
+```julia
+module MyPackage
+
+export a
+
+const NONEXPORTED_PUBLIC_NAMES = Symbol[:b, :c]
+
+"""
+`a` is exported, and it is part of the public API.
+"""
+function a end
+
+"""
+`b` is not exported, but it is part of the public API.
+"""
+function b end
+
+"""
+`c` is not exported, but it is part of the public API.
+"""
+function c end
+
+"""
+`d` is not exported, and it is private (internal).
+"""
+function d end
```

Comment:
Do you think it might be worth turning these into 1-line docstrings to save a few lines?

Reply:
Sure! What is the syntax for a one line docstring?